src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
I disagree with the "everyone uses it" argument. And even it would be right, why should really everyone use it.

The "symfony uses it" is invalid as well. (Who cares?)

I still it think is rather a personal preference. Again, from a logical point of view, integer is the type, and int is the alias. Even if it is the other way around in PHP, we are talking about documentation. (And I hate "standards" which some people call it a standard, because they think it is the only possible way to do things. It is just damn documentation)

Actually no, I don't. How is "standardclass" related to `MyClass[]`?